spector/frontend/src/css/addActivityPopup.css

```diff
@@ -0,0 +1,92 @@
+.buttonAdd, .buttonJoin{
+  cursor: pointer;
+  padding: 0px 30px;
+  display: inline-block;
+  margin: 5px 0px;
+  /*text-align: left;*/
+  line-height: 2em;
+  letter-spacing: 1.5px;
+  font-size: 1em;
+  outline: none;
+  position: relative;
+  font-size: 14px;
+  background-color: #bfb3ff;
+  outline: 2px solid white;
+  border-radius: 20px;
+  color: white;
+  border-radius: 15px 15px 15px 15px;
+  transition: all 0.3s;
+  width: 100%;
+}
+
+.buttonJoin {
+  border: 2px solid #aa99ff;
+  background-color: #ffffff;
+  width: 25%;
+  color: #000000;
+}
+  
+
+.buttonAdd:hover,
+.buttonAdd.hover {
+  border-color: #2980b9;
+}
+
+.popupHeader {
+    color:#00ff00;
+    text-align: center;
+    display: block;
+}
+
+.titleBox {
+    box-sizing:border-box;
+    width: 100%;
+}
+
+.descBox {
+    width: 100%;
+    box-sizing: border-box;
+    display:block;
+    resize:none;
+}
+
+/* TODO transfer and clean up styles*/
+.modal {
+  font-size: 12px;
+  /* padding: 15px; */
+  padding: 15px 15px 5px 15px;
+}
+.modal > .header {
+  width: 100%;
+  border-bottom: 1px solid gray;
+  font-size: 18px;
+  text-align: center;
+  padding: 5px 1px;
+}
+.modal > .content {
+  width: 100%;
+  padding: 10px 1px;
+}
+.modal > .actions {
+  margin: auto;
+  text-align: center;
+}
+  
+  /* X close button */
+.modal > .close {
```

Comment:
The CSS files are global. So as long as a CSS file is imported in one JS file, you can use its styles in any of the other JS files, without needing to import the CSS file.

So if these popup styles will be the same as in activityPopup.css then we don't need them to be duplicated. 

But if they differ then
- give them a unique name like `.Create-modal`
- or use a css module so the styles are local
```js
// Using a CSS module
import styles from "css/addActivityPopup.css";

<button className={styles.buttonAdd}>
    Add a New Event...
</button>
```